README.rst

```diff
@@ -9,6 +9,8 @@ Overview
 
 **Warning:** Subject to frequent updates
        This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+       
+This guide uses MUST and SHOULD, etc as defined in `RFC 2119 <https://datatracker.ietf.org/doc/html/rfc2119>`_. 
```

Comment:
```suggestion
This guide uses MUST, MUST NOT, SHOULD, SHOULD NOT and MAY as defined in `RFC 2119 <https://datatracker.ietf.org/doc/html/rfc2119>`_. 
```

I would mention all key words here instead of just saying `etc`. But that's just my personal opinion.